sw/airborne/firmwares/rotorcraft/guidance/guidance_indi_hybrid.c

```diff
@@ -86,6 +103,8 @@ float nav_max_speed = NAV_MAX_SPEED;
 
 /*Boolean to force the heading to a static value (only use for specific experiments)*/
 bool take_heading_control = false;
+/*Boolean to activate moving rope landing controller*/
+float approaching_rope = 0.0;
```

Comment:
I think using a float as a boolean is not really nice

Reply:
yes I agree. Because of how to test if it is on or not. In simulation it seems to work but it is not elegant. Changing it to boolean means that I have to delete the gain scheduling I previously did because it is used in mathematical operations. Which means that if we will need it again in the future (e.g. outdoor flight) I would have to rewrite it again. Shall I still make the conversion?

Reply:
changed to boolean